stock_picking_eco_tag/report/stock_picking_eco_tag_template.xml

```diff
@@ -15,7 +15,8 @@
                                     <div style="padding-top: 5px;">
                                         <p class="mb-0" style="font-size: 10px">Product: <t t-esc="move_line.product_id.name"/></p>
                                         <p class="mb-0" style="font-size: 10px">Var / Cat: <t t-esc="move_line.product_id.categ_id.name[3:]"/></p>
-                                        <p class="mb-0" style="font-size: 10px">Net weight: <t t-esc="move_line.qty_done"/> <t t-esc="move_line.product_id.uom_id.name"/></p>
+                                        <t t-set="decimal_precision" t-value="request.env['decimal.precision'].precision_get('Product Unit of Measure')"/>
```

Comment:
Is it possible to place this line outside `t-foreach` loop?

Reply:
Done